app/views/components/_guest_nav_drawer.html.erb

```diff
@@ -1,2 +1,3 @@
-<!-- <a href="/login" class="button&#45;login usa&#45;button usa&#45;button&#45;outline&#45;inverse">Login</a> -->
-<%= link_to 'Login', login_path, class: 'usa-button usa-button-outline-inverse' %>
+<% if request.path != '/login' %>
```

Comment:
very logic-y for a view? I guess if the thing will get tossed out, then no big deal?